### PR TITLE
Grouped Context Menus

### PR DESCRIPTION
**Pull request by neelip**
image

### CODE REVIEW

**Comment by nturinski** [comment]
Could you provide screenshots of what the new menus look like?  I think that'll make this a lot easier to review.

File: package.json [hunk 1]
```diff
@@ -532,332 +532,412 @@
                 {
                     "command": "azureDatabases.openInPortal",
                     "when": "view == azureDatabasesExplorer && viewItem == azureextensionui.azureSubscription",
-                    "group": "1@2"
+                    "group": "2@1"
```
Comment (ejizba) [inline package.json:535]: I like having "Open in Portal" and "Refresh" as a part of the same group, like [I did](https://github.com/microsoft/vscode-azureresourcegroups/blob/main/package.json#L144) in the rg extension. I consider the group to be "common commands across multiple nodes/extensions that are simple but not _that_ useful"

**Review by ejizba — APPROVED** [review]
FYI @neelip you'll get a faster response from us if you post a new comment rather than edit the description because it sends another email. I happened to see the edit this time, but won't always check

File: package.json (hunk 1 above)
Comment (nturinski) [reply package.json:535]: I agree with Eric.  Plus having a totally separate group for just "Open in Portal" looks a bit clunky to me.

**Comment by nturinski** [comment]
Not a requirement, but I find that this helps keep the `package.json` cleaner.  https://github.com/microsoft/vscode-azurestaticwebapps/blob/main/package.json#L310-L323

By using a Regex, you don't need multiple declarations of .refresh commands.  By grouping it as 9, you're guaranteeing that it'll be at the bottom of the list (unless for some reason we ever hit more than 9 groups 🥴)

**Pull request merged**